service/instance.py

```diff
@@ -234,10 +232,8 @@ def suspend_instance(esh_driver, esh_instance,
     """
     _permission_to_act(identity_uuid, "Suspend")
     if reclaim_ip:
-        remove_ips(esh_driver, esh_instance, identity_uuid)
+        remove_floating_ip(esh_driver, esh_instance, identity_uuid)
```

Comment:
With this change, we no longer need the periodic floating ip task to run as often, we could have it run once a week or less, for instances that are nuked outside of atmosphere

Reply:
agreed.